zebra/zebra_dplane.c

```diff
@@ -4485,6 +4673,13 @@ static void kernel_dplane_log_detail(struct zebra_dplane_ctx *ctx)
 				   dplane_op2str(dplane_ctx_get_op(ctx)),
 				   ipent.unique, ctx);
 	} break;
+
+	case DPLANE_OP_NEIGH_TABLE_UPDATE:
+		zlog_debug("Dplane neigh table op %s, ifp %s, family %s",
```

Comment:
unprotected debug

Reply:
are you su re ? this is in function log_detail ?

Reply:
oh, quite right - didn't notice the context of the new debug